pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -36,6 +37,12 @@ type NodeLabeller struct {
 	host              string
 	namespace         string
 	logger            *log.FilteredLogger
+	supportedCPUs     SupportedCPUsMap
+}
+
+type SupportedCPUsMap struct {
+	sync.RWMutex
+	CPUs []string
```

Comment:
Is this array filled from a go-routine which runs in parallel to the general initialization? It is ok to access maps and arrays from go-routines, if the go-routines fork off from the routine where the map was initially populated. That is probably the case?